### PR TITLE
Merge infrahub-git agent and task runner into task-worker for helm

### PR DESCRIPTION
**Pull request by wartraxx51**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # yamllint disable rule:line-length
 ---
 global:
+  infrahubTag: develop
```
Comment (wartraxx51) [inline helm/values.yaml]: i will delete this line after fix helm-lint

File: helm/Chart.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 apiVersion: v2
 name: infrahub
 description: A Helm chart to deploy Infrahub on Kubernetes
+icon: https://avatars.githubusercontent.com/u/118297816?s=200&v=4
```
Comment (fatih-acar) [inline helm/Chart.yaml]:
```suggestion
icon: https://github.com/opsmill/infrahub/raw/develop/frontend/app/public/favicons/logo512.png
```

?

**Comment by dgarros** [comment]
Thank you for taking care of it

File: helm/templates/infrahub-task-worker.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+              value: {{ $value | quote }}
+            {{- end }}
+          {{- with .Values.infrahubTaskWorker.infrahubTaskWorker.envFromExistingSecret }}
+          envFrom:
+            - secretRef:
+                name: {{ . }}
+          {{- end }}
+          image: {{ default .Values.global.imageRegistry .Values.infrahubTaskWorker.infrahubTaskWorker.imageRegistry }}/{{ .Values.global.infrahubRepository }}:{{ .Values.global.infrahubTag | default .Chart.AppVersion }}
+          imagePullPolicy: {{ default .Values.global.imagePullPolicy .Values.infrahubTaskWorker.infrahubTaskWorker.imagePullPolicy }}
+          resources: {}
```
Comment (fatih-acar) [inline helm/templates/infrahub-task-worker.yaml]:
```suggestion
          {{- with .Values.infrahubTaskWorker.resources }}
          resources: {{- toYaml . | nindent 12 }}
          {{- end }}
          {{- with .Values.infrahubTaskWorker.infrahubTaskWorker.securityContext }}
          securityContext: {{- toYaml . | nindent 12 }}
          {{- end }}
```

That was present in infrahub-git but was missing from infrahub-task-runner

File: helm/values.yaml [hunk 4]
```diff
@@ -53,45 +53,15 @@ neo4j:
...
-infrahubGit:
+# ----------- Infrahub Task Worker -----------
+infrahubTaskWorker:
   replicas: 2
   persistence:
     enabled: true
     size: 100Mi
     accessMode: ReadWriteMany
     storageClassName: nfs
-  resources: {}
```
Comment (fatih-acar) [inline helm/values.yaml:64]: We need to keep that `resources` line

**Review by fatih-acar — COMMENTED** [review]
LGTM with minor nits

**Review by fatih-acar — COMMENTED** [review]
LGTM but you need to discard the python_sdk change

**Pull request merged**